test/favorites.test.js

```diff
@@ -8,9 +8,7 @@ const db = require('knex')(require('../knexfile')['development']);
 
 describe("Favorites Endpoints", () => {
   before((done) => {
-    db.raw("TRUNCATE playlists_favorites restart identity;")
-    .then(() => db.raw("TRUNCATE playlists restart identity CASCADE;"))
-    .then(() => db.raw("TRUNCATE favorites restart identity CASCADE;"))
+    db.raw("TRUNCATE favorites restart identity CASCADE;")
```

Comment:
- Good call as far as shortening this down and making use of CASCADE to do the work for you.

- You should capitalize "RESTART IDENTITY" since they're keywords and not table names.